source/opt/debug_info_manager.h

```diff
@@ -128,6 +129,14 @@ class DebugInfoManager {
   uint32_t BuildDebugInlinedAtChain(uint32_t callee_inlined_at,
                                     DebugInlinedAtContext* inlined_at_ctx);
 
+  // Creates a DebugValue whose 'Local Variable' and 'Value' operands are
+  // |variable_id| and |value_id| and inserts it after |instr|.
```

Comment:
This needs to be updated.  It might create more than one, and is not clear on what the function is doing.

Try something like:

Generates a DebugValue instruction with value |value_id| for every local variable that is in scope and whose memory is |variable_id|.

Reply:
done